mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -224,6 +242,42 @@ def _load_mesh(runStreams):  # {{{
         refTopDepth, refLayerThickness  # }}}
 
 
+def _cache_velocity_climatologies(config, startDateClimo, endDateClimo,
+                                  inputFilesClimo, simulationStartTime,
+                                  variableMap, calendar):  # {{{
+    '''compute yearly velocity climatologies and cache them'''
+
+    variableList = ['avgNormalVelocity',
+                    'avgVertVelocityTop']
+
+    outputDirectory = build_config_full_path(config, 'output',
+                                             'mpasClimatologySubdirectory')
+
+    try:
+        os.makedirs(outputDirectory)
+    except OSError:
```

Comment:
If you try to write to a write-protected location, you'll get the appropriate OSError later on when you try to create files.  I feel like it's okay if this fails silently. 

Rather than adding a comment, I'll make sure this calls the utility function that does this same thing, and already has a comment.